src/leap/mail/mail.py

```diff
@@ -556,57 +555,29 @@ def insert_mdoc_id(_, wrapper):
         d = wrapper.create(self.store, notify_just_mdoc=notify_just_mdoc)
         d.addCallback(insert_mdoc_id, wrapper)
         d.addErrback(lambda f: f.printTraceback())
+        d.addCallback(self.cb_signal_unread_to_ui)
         return d
 
-    def add_raw_message(self, message, flags, date=None):
+    def cb_signal_unread_to_ui(self, result):
         """
-        Adds a message to this collection.
-
-        :param message: the raw message
-        :type message: str
-
-        :param flags: flag list
-        :type flags: list of str
+        Sends unread event to ui.
```

Comment:
Sure. The problem is that, in the client side, there's a message that says "N Unread messages in your Inbox"